### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by vidya-ram**
1. Use platform-native share(web api)
2. Fix bug of lazy loading on notification feed page. 
When the site updates page is loaded on a large screen, only the first page of notifications are shown. Second page is not loaded as there is no scroll activity, but a spinner appears permanently
3. Fix the padding of sub pages of project.
4. Change notification preferences choice to wrap on mobile
5. Fix the size and position of home icon on mobile and desktop
6. Change calendar strip to be full width

### CODE REVIEW

File: funnel/static/css/app.css [hunk 1]
```diff
@@ -1615,58 +1617,6 @@ a.loginbutton.hidden,
   padding-right: 0;
 }
 
-.tabs {
```
Comment (vidya-ram) [inline funnel/static/css/app.css:1618]: Moved to Basframe

File: funnel/templates/project.html.jinja2 [hunk 2]
```diff
@@ -69,6 +69,41 @@
 {% block contenthead %}{% endblock %}
 
 {% block baseheadline %}
+  {% if current_auth.user.is_site_editor %}
```
Comment (vidya-ram) [inline funnel/templates/project.html.jinja2:72]: Don't need feature project form in all sub pages of project

File: funnel/assets/js/update.js [hunk 3]
```diff
@@ -12,12 +12,13 @@ const Updates = {
...
     isEditor,
     headerHeight,
+    addReadMore,
   }) {
     Vue.use(VS2);
 
     const updateUI = Vue.component('update', {
       template: updateTemplate,
-      props: ['update', 'iseditor'],
+      props: ['update', 'iseditor', 'addreadmore'],
```
Comment (jace) [inline funnel/assets/js/update.js:21]: This isn't case sensitive?

**Review by jace — APPROVED** [review]

File: funnel/assets/js/update.js (hunk 3 above)
Comment (vidya-ram) [reply funnel/assets/js/update.js:21]: vue template variables(props) don't allow camel case

**Pull request merged**